api/cmd/api/main.go

```diff
@@ -63,6 +64,14 @@ func main() {
 		defer api.Cleanup()
 	}
 
+	{
+		// Populate Tables
+		initializer := initializer.New(api)
```

Comment:
should we skip this in `dev` mode :) ?

Reply:
we will address this later as discussed.